site-packages/integralstor_utils/zfs.py

```diff
@@ -1651,6 +1701,29 @@ def get_pool_usage(pool_name):
                 raise Exception(err)
             d['available_for_data'] = td
 
+        if d['available_for_data'] and d['used_by_data']:
```

Comment:
Are we sure that available_for_data and used_by_data keys will be there or shd we also add - "if available_for_data in d".. etc.